packages/redux-variants/variants.js

```diff
@@ -383,6 +412,24 @@ const filteredVariantsById = createSelector(
       filteredVariants = filteredVariants.filter(v => searchPredicate(v, query))
     }
 
+    // Indel and Snp filters.
+    filteredVariants = filteredVariants.filter(v => {
+      let splits = v.get('variant_id').split('-')
```

Comment:
I forgot to mention it, but we have ESLint and Prettier set up for a style guid... this doesn't pass the [prefer-const](https://eslint.org/docs/rules/prefer-const) rule.